daal4py/sklearn/ensemble/_forest.py

```diff
@@ -678,22 +686,22 @@ def predict(self, X):
         y : ndarray of shape (n_samples,) or (n_samples, n_outputs)
             The predicted classes.
         """
-
-        if not hasattr(self, 'daal_model_') or \
-                sp.issparse(X) or self.n_outputs_ != 1:
-            logging.info(
-                "sklearn.ensemble.RandomForestClassifier."
-                "predict: " + get_patch_message("sklearn"))
+        _patching_status = PatchingConditionsChain(
+            "sklearn.ensemble.RandomForestClassifier.predict")
+        _dal_ready = _patching_status.and_conditions([
+            (hasattr(self, 'daal_model_'), "onedal model was not trained"),

```

Comment:
**onedal** should be change to **oneDAL** Please change it in all places. Also somewhere before first using oneDAL in documentation or log file full name **Intel® oneAPI Data Analytics Library (oneDAL)** must be used at least once.

Reply:
Next message was added before all patching debug messages:
```python
logging.debug(
    f'{self.scope_name}: patching debugging is enabled to track usage of'
    ' Intel® oneAPI Data Analytics Library (oneDAL)')
```